src/factories/workflow-specific/PIM_WorkflowFactory.sol

```diff
@@ -0,0 +1,187 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+// Internal Interfaces
+import {IOrchestratorFactory_v1} from
+    "src/factories/interfaces/IOrchestratorFactory_v1.sol";
+import {IOrchestrator_v1} from
+    "src/orchestrator/interfaces/IOrchestrator_v1.sol";
+import {IFM_BC_Bancor_Redeeming_VirtualSupply_v1} from
+    "@fm/bondingCurve/interfaces/IFM_BC_Bancor_Redeeming_VirtualSupply_v1.sol";
+import {IPIM_WorkflowFactory} from
+    "src/factories/interfaces/IPIM_WorkflowFactory.sol";
+
+// External Interfaces
+import {IERC20} from "@oz/token/ERC20/IERC20.sol";
+
+// External Implementations
+import {ERC20Issuance_v1} from "src/external/token/ERC20Issuance_v1.sol";
+
+// External Dependencies
+import {Ownable} from "@oz/access/Ownable.sol";
+
+contract PIM_WorkflowFactory is Ownable, IPIM_WorkflowFactory {
```

Comment:
Second general question: Do we want or need MetaTx support here? Not sure how beneficial it is, as the user needs to approve the collateral token anyways I'd assume, so it's not a fully gas-less one, unless the collateral token has a `permit` function.

Reply:
Good question, I just added support so that we have it. If anyone thinks this is a bad idea, lmk